test_common/harness/imageHelpers.cpp

```diff
@@ -286,6 +286,7 @@ uint32_t get_pixel_size(const cl_image_format *format)
 
         case CL_FLOAT:
             return get_format_channel_count(format) * sizeof(cl_float);
+        case CL_UNORM_INT_101010_2: return 4;
```

Comment:
Nitpick: it'd be more consistent with the rest of the code in this function to move this line above, underneath the case statement for `CL_UNORM_INT_101010`.